### PR TITLE
use gif for building-grid texture

### PR DESCRIPTION
**Pull request by meetar**

### CODE REVIEW

File: eraser-map.yaml [hunk 1]
```diff
@@ -403,6 +403,9 @@ textures:
             winery: [792, 216, 36, 38]
             wings-joint: [756, 216, 36, 38]
             zoo: [612, 144, 36, 38]
+    grid:
```
Comment (nvkelso) [inline eraser-map.yaml]: Please use a more descriptive name here (like building-wall-grid or similar).

File: eraser-map.yaml [hunk 2]
```diff
@@ -806,29 +809,28 @@ styles:
         base: polygons
         mix: [hsv, scale-buildings]
         texcoords: true
+        material:
+            diffuse:
+                texture: grid
```
Comment (bcamper) [inline eraser-map.yaml]: Need to update this to reflect texture name

**Comment by bcamper** [comment]
@meetar can we to convert to PNG (since ES needs fixes for GIF)?

**Comment by nvkelso** [comment]
For documentation, should all facades be defined using a PNG/GIF texture as opposed to writing a fragment shader?

**Comment by bcamper** [comment]
@nvkelso which facades do you mean? All (and/or other specific ones in this style)? All in general? In this case, it is a pattern (a grid) that aliases badly when scaled, and can be easily re-created in a texture (actually the texture gets dynamically blended with the wall color, here: https://github.com/tangrams/eraser-map/pull/52/files#diff-939bb555547386cbe68c2ce87237df85R830). So it depends on the nature of the pattern and the other visual effects as to whether a texture, shader, or both is most appropriate.

**Comment by bcamper** [comment]
(By using a texture we get hardware mipmapping, which automatically scales patterns like a grid well across zoom levels.)

**Comment by bcamper** [comment]
Or @blair1618 is going to look at enabling GIF support for ES, which we need anyway... so.. either way :)

**Comment by nvkelso** [comment]
@bcamper Thanks for the background / documentation :)

**Comment by matteblair** [comment]
GIF images are working in tangram-es, just had to update stb_image to fix a bug in GIF transparency.

**Comment by sensescape** [comment]
looks good, thank you : )

**Pull request merged**